app/_hub/nonamesecurity/nonamesecurity-kongprevention/overview/_index.md

```diff
@@ -0,0 +1,125 @@
+
+### Performance Benchmarks 
+[Kong Enterprise 2.8.1.1 with Noname 3.1](https://docs.nonamesecurity.com/v320/docs/kong-performance-results)
+
+
+## How to install
+
+### The Noname blocking and prevention plugin for Kong 
+
+[Noname Security Kong Prevention install and configuration Documentation](https://docs.nonamesecurity.com/docs/kong-prevention)
+
+1. Create the Integration Profile
+
+In the Noname UI, navigate to Settings > Integrations > Prevention.
+Select Add Integration and select the Kong tile to create an integration profile.
+Download the Zip file, and copy it to your Kong machine.
+Select Next.
+Enter the Kong Admin API URL.
+Enter key-value sets for the Authentication headers to enable noname to authenticate with Kong.
+Select Next.
+Provide an alias for the integration.
+Select Finish to save the integration.
+
+2. Install the LuaRocks module
+
+Upload and activate the noname prevention plugin to kong using the following steps: In your Kong machine CLI shell Navigate to the location of the copied Zip file and unzip the file.
+Run the following command to install the plugin:
+
+
+```shell
+luarocks make
+
+```
+
+3. Update the Kong plugins configuration environment variable 
+
+Add the noname-prevention plugin to the kong configuration file (usually /etc/kong/kong.conf).
+
+    Open the file, find the plugins, and add noname-prevention. For example:
+    
+```yaml
+plugins = bundled,noname-prevention
+```
+
+4. Restart Kong
+
+After LuaRocks is installed, restart Kong before enabling the plugin
+
+```shell
+kong restart
+```
+
+### Install via a Dockerfile to create a custom kong docker image with both of the noname kong plugins preinstalled
+
+```yaml
+FROM kong/kong-gateway:3.4
+
+USER root
+
+RUN \apt-get update && \apt-get install unzip -y  
+
+WORKDIR /usr/kong/noname
+
+RUN apt update && apt-get install -y build-essential git curl unzip
+
+RUN bash -c 'mkdir -pv {nonamesecurity,prevention}'
+
+COPY ./noname-security-kong-policy.zip nonamesecurity/noname-security-kong-policy.zip
+
+RUN unzip nonamesecurity/noname-security-kong-policy.zip -d nonamesecurity && rm nonamesecurity/noname-security-kong-policy.zip
+
+COPY ./noname-security-kong-prevention.zip prevention/noname-security-kong-prevention.zip
+
+RUN unzip prevention/noname-security-kong-prevention.zip -d prevention && rm prevention/noname-security-kong-prevention.zip
+
+RUN cd nonamesecurity && luarocks make
+
+RUN cd prevention && luarocks make
+
+USER kong
+```
+
+Note the above Dockerfile installs both the noname traffic source for machine learning plugin and the noname prevention plugin for automated blocking. 
+
+## Using the Noname Security Kong Prevention plugin
+
+Configure the plugin
+```shell
+$ curl -X POST http://KONG_HOST:8001/services/<your-kong service-id>/plugins/ \
+--data "name=noname-prevention"
+```
```

Comment:
```suggestion
1. Configure the plugin: 
    ```shell
    curl -X POST http://localhost:8001/services/{serviceNameorId}/plugins \
       --data "name=noname-prevention"
    ```
```
We use `localhost` as the default host value in the our docs.